### PR TITLE
[Website] update PULL_REQUEST_TEMPLATE.md

### PR DESCRIPTION
**Pull request by bullet03**
- update ADD_LOGO.md with new info about hasNav, hasLink
- update CONTRIBUTE.md (how to add links to JS, how to add new logo, how to add new case study, fix typos, update structure)
- add CASE_STUDY_TEMPLATE.md which can be used as a template when creating new case-study
- add ADD_CASE_STUDY.md to be used as a recommendation how to fill new case-study
------------------------

Thank you for your contribution! Follow this checklist to help us incorporate your contribution quickly and easily:

 - [ ] [**Choose reviewer(s)**](https://beam.apache.org/contribute/#make-your-change) and mention them in a comment (`R: @username`).
 - [ ] Mention the appropriate issue in your description (for example: `addresses #123`), if applicable. This will automatically add a link to the pull request in the issue. If you would like the issue to automatically close on merging the pull request, comment `fixes #<ISSUE NUMBER>` instead.
 - [ ] Update `CHANGES.md` with noteworthy changes.
 - [ ] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

See the [Contributor Guide](https://beam.apache.org/contribute) for more tips on [how to make review process smoother](https://beam.apache.org/contribute/get-started-contributing/#make-the-reviewers-job-easier).

To check the build health, please visit [https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md](https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md)

GitHub Actions Tests Status (on master branch)
------------------------------------------------------------------------------------------------
[Build python source distribution and wheels](https://github.com/apache/beam/actions?query=workflow%3A%22Build+python+source+distribution+and+wheels%22+branch%3Amaster+event%3Aschedule)
[Python tests](https://github.com/apache/beam/actions?query=workflow%3A%22Python+Tests%22+branch%3Amaster+event%3Aschedule)
[Java tests](https://github.com/apache/beam/actions?query=workflow%3A%22Java+Tests%22+branch%3Amaster+event%3Aschedule)
[Go tests](https://github.com/apache/beam/actions?query=workflow%3A%22Go+tests%22+branch%3Amaster+event%3Aschedule)

See [CI.md](https://github.com/apache/beam/blob/master/CI.md) for more information about GitHub Actions CI.

### CODE REVIEW

File: website/CASE_STUDY_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,97 @@
+---
+title: "Case study title" 
+name: "Company/project name"
+icon: /images/logos/powered-by/company_name.png
+hasNav: true
```
Comment (AKosolapov) [inline website/CASE_STUDY_TEMPLATE.md:5]: @bullet03 can you check if this parameter is required?

Comment (bullet03) [reply website/CASE_STUDY_TEMPLATE.md:5]: added to the ADD_LOGO.md

**Comment by bullet03** [comment]
@pabloem, @aijamalnk, could you, please, help with review of PR?

**Comment by pabloem** [comment]
LGTM!

**Pull request merged**